tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -425,6 +425,30 @@ public static class Tags
         /// </summary>
         internal const string AppSecJson = "_dd.appsec.json";
 
+        /// <summary>
+        /// Ruleset file version, string satisfying the regular expression: [0-9]+\.[0-9]+\.[0-9]+
+        /// </summary>
+        internal const string AppSecRuleFileVersion = "_dd.appsec.event_rules.version";
+
+        /// <summary>
+        ///  String-serialized JSON array, each item being a map containing:
+        ///  Error(e) - the error string.
+        ///  Rules(r) - an array of rules which failed to load with this error.
+        /// </summary>
+        internal const string AppSecRuleFileErrors = "_dd.appsec.event_rules.errors";
+
+        /// <summary>
+        /// Version of the waf
+        /// </summary>
+        internal const string AppSecWafVersion = "_dd.appsec.waf.version";
+
+        /// <summary>
+        ///  String-serialized JSON array, each item being a map containing:
+        ///  Error(e) - the error string.
+        ///  Rules(r) - an array of rules which failed to load with this error.
+        /// </summary>
+        internal const string AppSecWafInitRuleErrors = "_dd.appsec.event_rules.errors";
```

Comment:
This tag `_dd.appsec.event_rules.errors` is the same one used in `AppSecRuleFileErrors` above. Was that intentional?